security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -38,6 +38,8 @@ The following advisory is available for the OpenShift Compliance Operator 1.4.0:
 [id="compliance-operator-1-4-0-bug-fixes"]
 === Bug fixes
 
+* Previously, on a cluster with Microsoft Windows nodes, some rules will FAIL after auto remediation is applied because the windows nodes were not skipped by the compliance scan. With this release, Microsoft Windows nodes are correctly skipped when remediation is applied. (link:https://issues.redhat.com/browse/OCPBUGS-7355[*OCPBUGS-7355*])
```

Comment:
Maybe better to use "Windows nodes" instead of "Microsoft Windows nodes"?

Reply:
Thank you @xiaojiey these changes are completed.